# Service to get the next grasping parameters to try while learning a grasping model
---
float64[] nextP       # The next parameters to try
int64 ret
string msg
